demo/specs/mig+mps/sharing-demo-job-1.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   namespace: sharing-demo
-  name: sharing-demo-job
+  name: sharing-demo-job-1
```

Comment:
I suppose one question I have is whether we should also rename the quickstart pod?

Also, does this require a readme update?

Reply:
This file was never meant to be run. It was just for showing during the demo. The name of the job is the same because in the demo I flip screens and show how the spec got expanded.

Reply:
There's no such an example in  the `quickstart` folder. The README in `mig+msps` doesn't mention `sharing-demo-job-1.yaml`. Perhaps we can add a description later.

Reply:
This file is only here because it's what I started off showing here:
https://youtu.be/1QfShSQLsbs?si=9rFmRUMi-e_puwO2&t=1128

Its not in the README because by the time I get to the README in the demo I don't need this file anymore.

Reply:
If you find it confusing to have in here but never meant to be run, I'd just remove it rather than update it.

Reply:
It could cause confusion to users. I've removed the file.

Reply:
Is the folder name `mig+mps` misleading too? There's a `mig+ts` configuration in the example. How about renaming the folder to `mig`?

Reply:
The purpose is to demonstrate mig+mps, the other configs are for comparison.